lms/validation/_lti_launch_params.py

```diff
@@ -17,14 +18,20 @@ class LaunchParamsSchema(PyramidRequestSchema):
     class URLSchema(Schema):
         """Schema containing only validation for the return URL."""
 
+        class Meta:
+            """Allow other values, as we are only here to validate the URL."""
+
+            unknown = EXCLUDE
+
         launch_presentation_return_url = fields.URL()
 
     resource_link_id = fields.Str(required=True)
     launch_presentation_return_url = fields.Str()
+    lti_version = fields.Str(validate=OneOf(["LTI-1p0"]), required=True)
```

Comment:
One of... this exact value

Reply:
Yeah it's amazing. And we're required to fail if it's not there